tests/test.js

```diff
@@ -438,4 +438,155 @@ test('NbsDecoder.getPackets() returns the last packet of each type when given a
   });
 });
 
+
+test('NbsEncoder constructor throws for invalid arguments', () => {
```

Comment:
Let's move these tests to a separate file, something like `test_encoder.js`. Will need to update the test script in `package.json`.

Reply:
Tests for encoder and decoder are now separate files.